src/client/components/fields/Date.tsx

```diff
@@ -15,6 +15,15 @@ const Date: React.FC<DateProps> = ({
   readOnly,
   isRequired,
 }) => {
+  const readOnlyProps = readOnly
```

Comment:
maybe these props could be useful in other fields, you could extract this

Reply:
Agreed!